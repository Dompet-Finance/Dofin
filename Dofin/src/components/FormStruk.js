import React, {Component} from 'react';
import {
  TextInput,
  View
} from 'react-native';

class FormStruk extends Component {
  constructor(props){
    super(props)
    this.state = {}
  }
  render(){
    return (
      <View>
        <TextInput value={"Nominal"}/>
        <TextInput value={"Item"}/>
        <TextInput value={"Kategori"}/>
        <TextInput value={"Lokasi"}/>
        <TextInput value={"Tanggal"}/>
      </View>
    )
  }
}

export default FormStruk
