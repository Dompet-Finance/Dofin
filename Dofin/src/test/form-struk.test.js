import React from 'react';
import renderer from 'react-test-renderer'
import {
  View,
  TextInput,
  Button
} from 'react-native';

import FormStruk from '../components/FormStruk.js'

describe('form struk testing', () => {
  test('it should render successfull', () => {
    const formStruk = renderer.create(
      <FormStruk />
    ).toJSON();
    expect(formStruk).toMatchSnapshot();
  })

  test('it should simulate click save on button', () => {
    const saveStruk = renderer.create(
      <Button/>
    )
  })
})
