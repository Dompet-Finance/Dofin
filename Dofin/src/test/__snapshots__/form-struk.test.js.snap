// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form struk testing it shoul render successfull 1`] = `
<View>
  <TextInput
    value="Nominal"
  />
  <TextInput
    value="Item"
  />
  <TextInput
    value="Kategori"
  />
  <TextInput
    value="Lokasi"
  />
  <TextInput
    value="Tanggal"
  />
</View>
`;
